public/app/plugins/datasource/loki/queryUtils.ts

```diff
@@ -304,6 +305,30 @@ export function getStreamSelectorsFromQuery(query: string): string[] {
   return labelMatchers;
 }
 
+export function removeTrailingPipelines(query: string): string {
```

Comment:
Would you mind adding a few unit tests for this function in `queryUtils.test`? Also, could this be doing something similar as using https://lodash.com/docs/4.17.15#trimEnd ?

Reply:
Also I'm not sure if this is the desired behavior, but when there's not trailing pipe it seems to return an empty query.

![imagen](https://user-images.githubusercontent.com/1069378/225053427-bc9a4ed6-d408-42b7-a6c8-d371dd216cd1.png)

Reply:
Good suggestion! I'll use https://lodash.com/docs/4.17.15#trimEnd to make it more simple.